components/VideoEmbedPreview/VideoEmbedPreview.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import { Meta } from "@storybook/react";
+import VideoEmbedPreview from "./VideoEmbedPreview";
+
+export default {
+  title: "VideoEmbedPreview",
+  component: VideoEmbedPreview,
+  args: {
+    video: {
+      url: "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
+    },
+  },
+  parameters: {
+    nextjs: {
+      appDirectory: true,
+    },
+  },
+} as Meta<typeof VideoEmbedPreview>;
+
+export const Default = {};
```

Comment:
Maybe we can create other stories with [`viewport`](https://storybook.js.org/docs/react/essentials/viewport), that way we can change the dimensions of the canvas.

For example, the story `Default` applies a viewport of 'responsive', we can export others like, Mobile, Tablet, Etc.

```ts
export const Mobile : Story = {
  parameters: {
    viewport: {
      defaultViewport: "mobile1",
    }
  }
}
```

Available viewports are: `responsive`, `mobile1`, `mobile2`, `tablet`, `desktop`

Reply:
@alexxuss1996 you may add just one of those if you prefer, there are named like this:

![image](https://user-images.githubusercontent.com/78808163/233394500-fef8a03d-b199-4fc7-85a7-eb1cc2c08cd4.png)

Reply:
@TonyMckes Thank you, I already figured it out!